arkouda/pdarrayIO.py

```diff
@@ -492,7 +369,7 @@ def load_all(path_prefix: str) -> Mapping[str, Union[pdarray, Strings, Categoric
 
     See Also
     --------
-    save_all, load, read_hdf, read_all
+    save_all, load, read, read
```

Comment:
Is read meant to be listed twice?

Reply:
Ah, good catch -- I did a find and replace on both `read_hdf` and `read_all`, so that is why there is a double here (and probably elsewhere)